integration_tests/src/main/python/aqe_test.py

```diff
@@ -0,0 +1,145 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from pyspark.sql.functions import when, col
+from pyspark.sql.types import *
+from asserts import assert_gpu_and_cpu_are_equal_collect
+from data_gen import *
+from marks import ignore_order, allow_non_gpu
+from spark_session import with_cpu_session
+
+_adaptive_conf = { "spark.sql.adaptive.enabled": "true" }
+
+# Dynamic switching of join strategies
+# Dynamic coalescing of shuffle partitions
+# Dynamically Handle Skew Joins
+
+_adaptive_coalese_conf = copy_and_update(_adaptive_conf, {
```

Comment:
are we calling this out explicitly just to make sure enabled everywhere or was this off in one of distributions?  if just to make sure perhaps we want to include the other ones like  spark.sql.adaptive.skewJoin.enabled

Reply:
This was some old code, I think we should handle this configuration testing more so in https://github.com/NVIDIA/spark-rapids/issues/6253. I have removed it for now